setup.cfg

```diff
@@ -22,7 +22,7 @@ replace = Version="{new_version}"
 addopts =
 	--strict
 	--tb=native
-	tests/
```

Comment:
Taking this out is causing Travis CI to find all written tests, including deprecated ones. Might want to place this back in.